src/components/themed/TransactionListTop.tsx

```diff
@@ -581,10 +581,9 @@ export class TransactionListTopComponent extends React.PureComponent<Props, Stat
     }
 
     // Handle StakePlugin staking
-    if (stakePolicies.length === 1) {
-      const [stakePolicy] = stakePolicies
-      const { stakePolicyId } = stakePolicy
-      const stakePlugin = getPluginFromPolicyId(stakePlugins, stakePolicy.stakePolicyId, {
+    if (Object.keys(stakePositionMap).length === 1) {
+      const stakePolicyId = Object.keys(stakePositionMap)[0]
+      const stakePlugin = getPluginFromPolicyId(stakePlugins, stakePolicyId, {
```

Comment:
Please revert this change in behavior, which seems broken. Before, we would bypass the selection page if there was only one option on that page. Now, we bypass the selection page if you have staked funds with a single provider. This means:

1. If there is only one option, but you haven't used it yet, you still get taken to the selection page.
2. If there are multiple options, but you've already staked with one, you can't select another.

Unless there's some other reason I'm not seeing, this specific change looks like a bug.